.contributing/the-process.md

```diff
@@ -75,16 +75,49 @@ Create: 
 
     -   A User Flow 
 
-        -   Step 1: Make a doc named: "x. *Use Case* < User Flow < *Section* (CL)"
+        1. Create a folder to hold all files needed to create a user flow
+            - Add title: "x. *Use Case* - User Flow files - *Section* (CL)"
+
+        2. Inside folder created in step 1, create a doc to outline action steps for the use case. 
+            - Add title: "x. *Use Case* - User Flow - *Section* (CL)"
+            - Add template to doc: 
+                -   A leaguer who...
+                    -   (context) 
+                -   wants to...
+                    -   (goal) 
+                -   so they...
+                    -   (action steps)
+            - After adding action steps, create "input task" in asana
+
+
+        3. Create lofi wireframe
+            - Sketch lofi wireframe on paper
+            - Create a google drawing to get input on sketch
+                - Add title: "x. *Use Case* - Lofi Sketch - *Section* (CL)"
+            - Take a photo and upload it to google drawing doc
+            - Add any helpful comments, e.g reasoning for design decisions, specific questions about design
+            - Create an "input task" in asana
+            - After 24 hr input time has ended, resolve any conversations (use real time chat if need be)
```

Comment:
What do you think if we said: 
At standdown: 
- resolve any conversations (add RTCs for the next day if need be) 
At next standup: 
- remind everyone the doc will be sealed at EOD 
At next standdown: 
- give everyone a final warning 
- resolve any conversations (use real time chat if need be) 
- declare the doc sealed

Reply:
love 'declare the doc sealed' 😆

Reply:
and i like the suggestion!

Reply:
imp-lem-ent-ing :)

Reply:
you-da-man!